### PR TITLE
Update torch-xpu-ops commit pin

### PR DESCRIPTION
**Pull request by toyxu**
Update the torch-xpu-ops commit to [01f4e29](https://github.com/intel/torch-xpu-ops/commit/01f4e293fa39818bd0d018e9bb82d4e2cf54be48), includes:
- Improve XPU operator coverage
- Fix `Werror=comments` relevant building issues

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang @aakhundov

### CODE REVIEW

**Comment by EikanWang** [comment]
@xytintel , pls. help update the PR description.

**Review by EikanWang — APPROVED** [review]

**Review by EikanWang — APPROVED** [review]

**Review by atalman — APPROVED** [review]
lgtm

**Comment by EikanWang** [comment]
@xytintel , pls. refine the test cases.

File: test/inductor/test_decompose_mem_bound_mm.py [hunk 1]
```diff
@@ -418,4 +418,6 @@ def test_check_device(self):
 
 
 if __name__ == "__main__":
-    run_tests()
+    # Intel GPU has not enable decompose_mem_bound_mm PASS yet
+    if not torch.xpu.is_available():
+        run_tests()
```
Comment (EikanWang) [inline test/inductor/test_decompose_mem_bound_mm.py:423]: It does not make sense to bypass the test file. Pls. revert the change.

File: test/inductor/test_torchinductor_opinfo.py [hunk 2]
```diff
@@ -349,8 +349,8 @@ def format_op(op):
     # a deconvolution forward propagation primitive
     "nn.functional.conv_transpose2d": {f32, f64},
     "nn.functional.conv_transpose3d": {f32, f64},
-    # rrelu not supported on XPU now
-    "nn.functional.rrelu": {f16, f32, f64},
+    # frexp not supported on XPU now
+    "frexp": {f16, f32, f64},
```
Comment (EikanWang) [inline test/inductor/test_torchinductor_opinfo.py:353]: Why do we remove `rrelu`? We have supported it?

**Review by EikanWang — CHANGES_REQUESTED** [review]

File: test/inductor/test_torchinductor_opinfo.py (hunk 2 above)
Comment (xytintel) [reply test/inductor/test_torchinductor_opinfo.py:353]: Yes

**Review by EikanWang — APPROVED** [review]
